clang/lib/CIR/Dialect/Transforms/TargetLowering/Targets/X86.cpp

```diff
@@ -90,17 +90,17 @@ static bool BitsContainNoUserData(Type Ty, unsigned StartBit, unsigned EndBit,
 }
 
 /// Return a floating point type at the specified offset.
-Type getFPTypeAtOffset(Type IRType, unsigned IROffset,
-                       const ::cir::CIRDataLayout &TD) {
-  if (IROffset == 0 && isa<SingleType, DoubleType>(IRType))
-    return IRType;
+Type getFPTypeAtOffset(Type irType, unsigned irOffset,
+                       const ::cir::CIRDataLayout &td) {
+  if (irOffset == 0 && isa<SingleType, DoubleType>(irType))
+    return irType;
 
   llvm_unreachable("NYI");
 }
 
 } // namespace
 
-class X86_64ABIInfo : public ABIInfo {
+class X8664AbiInfo : public ABIInfo {
```

Comment:
I object to removing the underscore from this name.  The generally accepted name for this architecture is `x86_64`.  People expect to see the underscore.

Reply:
Good catch. I hope the clang-tidy check is configurable enough (or provides some manual escape hatch) to fix this.

Reply:
Yea, there's a number of these types of changes. Needa figure out if we can get ChatGPT to sort through this 35k line diff and see if it can find things like this.